src/components/StudentFeedback.vue

```diff
@@ -23,9 +23,38 @@
         </p>
       </div>
     </div>
+    <div v-else>
+      <h1 class="s-h1">There is nothing for this module yet. Stay tuned.</h1>
+    </div>
```

Comment:
Great addition with the `v-if` and `v-else` to conditionally render this error-ish message